webapp/channels/src/components/threading/thread_viewer/thread_viewer.tsx

```diff
@@ -173,7 +174,24 @@ export default class ThreadViewer extends React.PureComponent<Props, State> {
     // scrolls to either bottom or new messages line
     private onInit = async (reconnected = false): Promise<void> => {
         this.setState({isLoading: !reconnected});
-        await this.props.actions.getPostThread(this.props.selected?.id || this.props.rootPostId, !reconnected);
+        const res = await this.props.actions.getPostThread(this.props.selected?.id || this.props.rootPostId, !reconnected, this.props.lastUpdateAt);
```

Comment:
I think we need to check if the result has a data field, or an error field. (see webapp/channels/src/packages/mattermost-redux/src/actions/threads.ts L303)

Reply:
Ok, then actions/websocket_actions.jsx#L849 is also wrong. I will correct that.